api/e2e/test/00_all_e2e_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
What's the reason for this build tag change?

Reply:
Ah, I'm using go 1.17 on my local machine and go 1.17 has introduced transition from `+build ...` tags to `go:build ...` format. ([Release Notes](https://golang.org/doc/go1.17)). 
Both tags are currently accepted so I'll just keep it here as we will eventually have to upgrade anyway.